lib/node/runner/checker_ballot_transaction.go

```diff
@@ -25,6 +28,41 @@ type BallotTransactionChecker struct {
 	ValidTransactions     []string
 	validTransactionsMap  map[string]bool
 	CheckTransactionsOnly bool
+	transactions          map[string]transaction.Transaction
+}
+
+func (b *BallotTransactionChecker) getTransaction(hash string) (tx transaction.Transaction, found bool, err error) {
```

Comment:
This `getTransaction` method has two feature.
One is to insert transaction into `b.transactions` and another is getting transaction.
I figure, it would be better to split it to two methods(insertTransaction, getTransaction).

Reply:
@Charleslee522 `transaction` is just for cache :)

Reply:
> @Charleslee522 `transaction` is just for cache :)

@spikeekips Okay, we can split it later :)